operations/reservations.md

```diff
@@ -217,6 +218,7 @@ Returns all reservations specified by any identifier, customer or other filter.
 | `ChildCount` | number | required | Count of children the reservation was booked for. |
 | `CustomerId` | string | required | Unique identifier of the [Customer](customers.md#customer) who owns the reservation. |
 | `CompanionIds` | array of string | required | Unique identifiers of [Customer](customers.md#customer)s that will use the resource. |
+| `VoucherId` | string | optional | Unique identifier of the [Voucher](services.md#voucher) that has been used to create reservation. |
```

Comment:
Should be somewhere higher, this logically belongs close to rate / company / travel agency.